api/src/main/java/com/openlattice/search/requests/Search.java

```diff
@@ -46,7 +46,7 @@ public Search(
             @JsonProperty( SerializationConstants.EXCLUDE_PROPERTY_TYPES ) Optional<Boolean> excludePropertyTypes,
             @JsonProperty( SerializationConstants.START ) int start,
             @JsonProperty( SerializationConstants.MAX_HITS ) int maxHits ) {
-        optionalKeyword = keyword.isPresent() ? Optional.of( keyword.get().trim() ) : Optional.empty();
+        optionalKeyword = keyword.map( String::trim );
```

Comment:
Does this end up keeping it Optional.empty if it is absent? I didn't even knew there was a map for optional

Reply:
yep! I didn't know about this one either